### DISCUSSION TITLE
I got an error

### DISCUSSION THREAD

**Discussion by Rojasundharam** [supabase/supabase#31404] [Questions]
Failed to match page sections: {"code":"PGRST202","details":"Searched for the function public.match_page_sections with parameters embedding, match_count, match_threshold, min_content_length or with a single unnamed json/jsonb parameter, but no matches were found in the schema cache.","hint":null,"message":"Could not find the function public.match_page_sections(embedding, match_count, match_threshold, min_content_length) in the schema cache"}

**Comment by GaryAustin1** [comment]
The error is pretty clear it can't find that function with those EXACT parameter names and types.

You would need to provide your call and your function definition for anyone to have a shot at guessing what is coded wrong.
